script/collections/brand-consolidation.json

```diff
@@ -75,15 +75,15 @@
     "sortBy": "order",
     "metadata": {
       "name": "Disability Benefits",
-      "icon": "fa-handshake-o hub-background-disability"
+      "icon": "fa-file-text-o hub-background-disability"
```

Comment:
It's probably too late in the game to benefit from it, but it was be awesome to see that reduced to just `hub-icon-disability` so we only have to change the FA icon in CSS.

Reply:
I think we should definitely do that eventually. I started off trying to do that, but I found that it was going to take me a little longer to make that work and get everything positioned correctly (because we have a bunch of positioning tweaks that are icon specific) than I was comfortable with for these changes.